src/SignalRServiceExtension/ServerlessHubContextAttribute.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+
+namespace Microsoft.Azure.WebJobs.Extensions.SignalRService
+{
+  /// <summary>
+  /// Customized settings to be passed into the serverless hub context.
+  /// </summary>
+  [AttributeUsage(AttributeTargets.Class)]
+  class ServerlessHubContextAttribute : Attribute, IConnectionProvider
```

Comment:
With `SignalRConnectionAttribute`, we can delete this class.

Reply:
ah right, I should not have reverted that commit

Reply:
done